index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Главная - Cat Energy</title>
+  </head>
+  <body>
+    <header class="page-header">
+      <a class="page-header__logo logo" href="index.html">
+        <img class="logo__image" src="images/logo-mobile.png" width="33" height="37" alt="Логотип Cat Energy.">
+        <img class="logo__text" src="images/logo-text-mobile.svg" width="101" height="18" alt="логотип мобильный.">
+      </a>
+      <nav class="page-header__main-nav main-nav main-nav--close">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть/закрыть меню</span>
+        </button>
+        <ul class="main-nav__list">
+          <li class="main-nav__item">
+            <a class="main-nav-link main-nav__link--active" href="index.html">Главная</a>
+          </li>
+          <li class="main-navitem">
+            <a class="main-nav__link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="main-nav__item">
+            <a class="main-nav__link" href="form.html">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="promo">
+        <h1 class="promo__title">Функциональное питание для котов</h1>
+        <p class="promo__info">Занялся собой? Займись котом!</p>
+        <img class="promo__image" src="images/index-can.png" width="280" height="270" alt="Упакова товара Cat Energy.">
+        <a class="promo__button button" href="form.html">Подобрать программу</a>
+      </section>
+      <section class="power-lines">
+        <h2 class="visually-hidden">Линейки питания</h2>
+        <ul class="power-lines__wrapper">
```

Comment:
Классы списков мы называем через слово list 
```suggestion
        <ul class="power-lines__list">
```